packages/core/src/navigation/navigation-item.element.scss

```diff
@@ -9,20 +9,32 @@
 :host {
   --color: inherit;
   --line-height: inherit;
+  --focus-outline: Highlight solid var(--focus-width);
+  --focus-width: #{$cds-alias-object-border-width-300};
   --font-size: inherit;
   --font-weight: inherit;
   --letter-spacing: inherit;
   --padding: inherit;
+  padding-left: var(--focus-width);
 }
 
 :host(:hover:not([disabled])),
 :host([_has-focus]) {
   background: #{$cds-alias-object-interaction-background-hover};
+  border: 1px solid var(--cds-alias-object-interaction-background-highlight);
 }
 
 :host(:active),
 :host([active]) {
   background: #{$cds-alias-object-interaction-background-active};
+  padding-left: 0;
+  border-left: var(--focus-width) solid var(--cds-alias-object-interaction-background-highlight);
+}
+
+:host(:active:focus),
+:host([_has-focus][active]) {
+  background: #{$cds-alias-object-interaction-background-hover};
+  border: 1px solid var(--cds-alias-object-interaction-background-highlight);
```

Comment:
Same here with `1px`

Reply:
Updated to var(--cds-alias-object-border-width-100)